src/renderers/shared/stack/reconciler/__tests__/ReactUpdates-test.js

```diff
@@ -1147,4 +1147,10 @@ describe('ReactUpdates', function() {
     ReactDOM.render(<App />, document.createElement('div'));
   });
 
+  it('unstable_batchedUpdates should return value from a callback', function() {
+    var result = ReactDOM.unstable_batchedUpdates(function() {
+      return 42
```

Comment:
Yes, though apparently it's only a warning so didn't fail CI (https://travis-ci.org/facebook/react/jobs/150690962#L285)